### PR TITLE
refactor: fix import path error

### PR DESCRIPTION
**Pull request by ttimochan**
你看看合不合适

### CODE REVIEW

**Comment by wibus-wee** [comment]
那其实我觉得可以把引入变成 `./icalingua-theme-telegram/...`?

File: install.sh [hunk 1]
```diff
@@ -6,9 +6,12 @@ if [ "$(uname)" == "Darwin" ]; then
    echo "Done."
 # Check if the system is Linux
 elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
-   cp ./src/* ~/.config/icalingua
+   cp -r ./src ~/.config/icalingua
    cp style.css ~/.config/icalingua
+   mv addon.js ~/.config/icalingua
```
Comment (wibus-wee) [inline install.sh:11]: I think it should be `cp` instead of `mv`. Using `mv` may lead to git conflict?

File: install.sh [hunk 2]
```diff
@@ -6,9 +6,12 @@ if [ "$(uname)" == "Darwin" ]; then
    echo "Done."
```
Comment (wibus-wee) [inline install.sh:6]: and you forgot to copy `addon.js` in Darwin.😂

**Review by wibus-wee — CHANGES_REQUESTED** [review]

File: install.sh (hunk 2 above)
Comment (ttimochan) [reply install.sh:6]: I don't know the macOS application profile directory, so I can only guess

**Comment by wibus-wee** [comment]
`src` -> `icalingua-theme-telegram`

**Review by wibus-wee — APPROVED** [review]

**Comment by wibus-wee** [comment]
tql

**Pull request merged**